.ibm/manual-tests/postgres/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+set -x
+
+HELM_IMAGE_NAME=backstage
+HELM_REPO_NAME=rhdh-chart
+HELM_REPO_URL="https://redhat-developer.github.io/rhdh-chart"
```

Comment:
Will this use the production helm chart?

Reply:
@rnapoles-rh we can change the helm chart for whatever we want change only changing the variables HELM_REPO_URL, HELM_REPO_NAME,  HELM_IMAGE_NAME

Reply:
Got it, thx